common/models/user.js

```diff
@@ -7,6 +7,41 @@ var config = require('../../server/config.json');
 var path = require('path');
 
 module.exports = function(User) {
+
+//send email again incase of unverified email
+User.triggerEmail = function(userId, cb) {
+  User.find({where: {id: userId}}, function(err, user)
+  {
+    const userObj = user[0];
+      var options = {
+      type: 'email',
+      to: userObj.email,
+      from: 'noreply@loopback.com',
+      subject: 'Thanks for registering.',
+      template: path.resolve(__dirname, '../../server/views/verify.ejs'),
+      redirect: '/verified',
+      user: userObj
+    };
+    userObj.verify(options, function(err, response) {
+      if (err) {
+        User.deleteById(userObj.id);
+        return next(err);
+      }
+      console.log('> verification email sent:', response);
+    });
+  })
+  cb(null, 'email triggered');
```

Comment:
I think the simplest option is what I described in https://github.com/strongloop-internal/scrum-asteroid/issues/130#issuecomment-299401881:

> Modify the REST API to accept redirect_url query argument so that set-password REST API call can redirect the user to a view page. Hopefully this can be achieved via an after-remote hook.

In other words, the code building the HTML email body (which contains the verification link) should include `redirect_url` in the link query string. I don't know where is this HTML code created.

The second step is to add "afterRemote" hook for "verify" method that will send HTTP redirect response when "redirect_url" was present in request query string.